tests/integration/Currency.test.js

```diff
@@ -14,25 +14,26 @@ describe('currency', () => {
     assert.ok(cur[currency.LAST_UPDATED] > 0);
     assert.ok(cur.BTC_USD > 0);
 
+    // disabled, because Mintlayer currency API doesn't currently support other fiat units
```

Comment:
We have separate API for ml currency rate
http://51.158.172.176:3000/getCurrentRate/ml/usd

Reply:
Correct, that's why it's disabled temporary, because this API doesn't support other fiat units.